pages.it/common/node.md

```diff
@@ -0,0 +1,24 @@
+# node
+
+> Piattaforma JavaScript Server-side (Node.js).
+> Maggiori informazioni: <https://nodejs.org>.
+
+- Esegue un file JavaScript:
+
+`node {{percorso/del/file}}`
+
+- Avvia una REPL (shell interattiva):
+
+`node`
+
+- Esegue il codice JavaScript che viene specificato come argomento:
+
+`node -e "{{codice}}"`
+
+- Mostra come risultato le versioni delle dipendenze di node:
+
+`node -p "process.versions"`
```

Comment:
This is something that, especially in this kind of project where the double curly brackets have a specific meaning, can be very misleading since one could be tempted to translate it too while it has actually to do with the internals of `node`.

I opened a PR to fix this: https://github.com/tldr-pages/tldr/pull/8899